server/views/partials/docs.njk

```diff
@@ -0,0 +1,44 @@
+
+{% macro drift() %}
```

Comment:
added these doc macros to show in multiple places.. we could migrate this to dev docs and have a link once we have some dev docs set up

Reply:
should probably run this through some LLM to tidy up the text 😅